content/en/docs/Configuration/p8s-jaeger-grafana/prometheus.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Prometheus"
+description: >
+  This page describes how to configure Prometheus for Kiali.
+---
+
+
+## Prometheus configuration
+
+Kiali *requires* Prometheus to generate the
+[topology graph]({{< relref "../../Features/topology" >}}),
+[show metrics]({{< relref "../../Features/details#metrics" >}}),
+[calculate health]({{< relref "../../Features/health" >}}) and
+for several other features. If Prometheus is missing or Kiali
+can't reach it, Kiali won't work properly.
+
+By default, Kiali assumes that Prometheus is available at the URL of the form
+`http://prometheus.<istio_namespace_name>:9090`, which is the usual case if you
+are using [the Prometheus Istio
+add-on](https://istio.io/latest/docs/ops/integrations/prometheus/#option-1-quick-start).
+If your Prometheus instance has a different service name or is installed to a
+different namespace, you must manually provide the endpoint where it is
+available, like in the following example:
+
+```yaml
+spec:
+  external_services:
+    prometheus:
+      # Prometheus service name is "metrics" and is in the "telemetry" namespace
+      url: "http://metrics.telemetry:9090/"
+```
+
+{{% alert color="success" %}}
+Notice that you don't need to expose Prometheus outside the cluster. It is
+enough to provide the Kubernetes internal service URL.
+{{% /alert %}}
+
+Kiali maintains an internal cache of some Prometheus queries to improve
+performance (mainly, the queries to calculate Health indicators). It
+would be very rare to see data delays, but should you notice any delays you may
+tune caching parameters to values that work better for your environment. These
+are the default values:
+
+```yaml
+spec:
+  external_services:
+    prometheus:
+      cache_enabled: true
+      # Per-query expiration in seconds
+      cache_duration: 10
+      # Global cache expiration in seconds. Think of it as
+      # the "reset" or "garbage collection" interval.
+      cache_expiration: 300
+```
+
+### Compatibility with Prometheus-like servers
+
+Although Kiali assumes a Prometheus server and is tested against it, there are
```

Comment:
Testing "against" something? Or testing "with" something? I think it should be you tested WITH Prometheus.

```suggestion
Although Kiali assumes a Prometheus server and is tested with it, there are
```

Reply:
Fine as-is.